n4js-libs/packages/n4js-cli/package.json

```diff
@@ -7,7 +7,7 @@
   "internalVersion": "2019-03-15T08:46:12Z",
   "description": "N4JS compiler and command line tools.",
   "scripts": {
-    "postinstall": "./npm-postinstall.js",
+    "postinstall": "node ./npm-postinstall.js",
     "build": "./npm-build.sh",
```

Comment:
Have you checked whether other package.json files contain similar cases? (shell scripts, .js-files launched via shebang)

Reply:
yes, I have checked it and could not find any